lib/MqttService/src/MqttService.cpp

```diff
@@ -163,9 +163,10 @@ void MqttService::process()
                 /* Authentication necessary? */
                 if (false == m_user.isEmpty())
                 {
-                    LOG_INFO("Connect to %s as %s with %s.", m_url.c_str(), m_user.c_str(), m_hostname.c_str());
+                    LOG_INFO("Connect to %s as %s : %s with %s.", m_url.c_str(), m_user.c_str(), m_password.c_str(), m_hostname.c_str());

```

Comment:
I would like to skip showing the password.

Reply:
I was just testing this because grepping user and password out of the string in settings did not work